swan/scripts/others/userconfig.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# Author: Danilo Piparo, Enric Tejedor, Pedro Maximino, Diogo Castro 2023
+# Copyright CERN
+# Here the user session is configured, including the home directory and 
+# the terminal.
+
+_log () {
+    if [[ "$*" == "ERROR:"* ]] || [[ "$*" == "WARNING:"* ]] || [[ "${JUPYTER_DOCKER_STACKS_QUIET}" == "" ]]; then
+        echo "$@"
+    fi
+}
+
+START_TIME_SETUP_SWAN_HOME=$( date +%s.%N )
+SETUP_SWAN_HOME_TIME_SEC=$(echo $(date +%s.%N --date="$START_TIME_SETUP_SWAN_HOME seconds ago") | bc)
+_log "user: $USER, host: ${SERVER_HOSTNAME%%.*}, metric: configure_user_env_swan_home.duration_sec, value: $SETUP_SWAN_HOME_TIME_SEC"
+
+# FIXME: Move to extensions.
+# Persist enabled notebook nbextensions
+SWAN_PROJECTS=$HOME/SWAN_projects
+NBCONFIG=$JPY_DIR/nbconfig
+mkdir -p $NBCONFIG
+LOCAL_NB_NBEXTENSIONS=$SWAN_PROJECTS/.notebook_nbextensions
+if [ ! -f $LOCAL_NB_NBEXTENSIONS ]; then 
```

Comment:
@diocas do we still need / want this?

Reply:
I would keep it while the old ui is available.